physical/postgresql/configure.go

```diff
@@ -0,0 +1,21 @@
+package postgresql
```

Comment:
Considering how this file only contains a single method, which itself is quite small, I'm not sure we gain much by putting it into it's own file. I would prefer it just be added to the bottom of the `postgresql.go` file